postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -139,6 +140,16 @@ class DBExplainError(Enum):
     # a truncated statement can't be explained
     query_truncated = "query_truncated"
 
+    # connection error may be due to a misconfiguration during setup
+    connection_error = 'connection_error'
+
+    # clients using the extended query protocol can't be explained due to
+    # the separation of the parsed query and raw bind parameters
+    extended_query_protocol = 'extended_query_protocol'
```

Comment:
Technically, this name is actually wrong. "extended_query_protocol" would have the same symptoms as prepared statements. In this case, the error should be something like `parameterized_query`.

Reply:
https://github.com/DataDog/integrations-core/pull/13224/commits/f73b6e33244b7b5db09ea529546c3bc5ed05e18a